### ISSUE TITLE
How filter SearchQuerySet?

### ISSUE THREAD

**Issue by Zverushko** [django-haystack/django-haystack#1450]
Hello! 
from haystack.query import SearchQuerySet
SearchQuerySet() - return all results

but i want have SearchQuerySet for any set of products in catalog for faceting

i will add 
id = indexes.IntegerField(model_attr='id')
in ProductIndex
and use SearchQuerySet().filter(id__in=[list of products_id])
right?

thx!

**Comment by LegoStormtroopr** [comment]
Yes. That is correct. You can use a Haystack `SearchQuerySet` in a very similar fashion to a Django `QuerySet`
